### ISSUE TITLE
"SPC w S" and "SPC w V" do not work anymore

### ISSUE THREAD

**Issue by marcoieni** [MarcoIeni/intelli-space#4]
All the actions prefixed with `VimWindow` do not work anymore.
I fixed almost all the actions under `SPC w` by replacing `VimWindow` actions with vim keys, for example `VimWindowDown` was replaced with `<C-w>j`.

The problem is with `SPC w S` and `SPC w V`.

https://github.com/MarcoIeni/intelli-space/blob/c0e2853a9526c202fe84cb29628cac3ff5dcc2f4/spacemacs/windows.vim#L51-L57

When I replace them with the following code it doesn't work.
```
" Split window below and focus
nnoremap <leader>wS    :action SplitHorizontally<CR><C-w>j
vnoremap <leader>wS    <Esc>:action SplitHorizontally<CR><C-w>j

" Split window right and focus
nnoremap <leader>wV    :action SplitVertically<CR><C-w>l
vnoremap <leader>wV    <Esc>:action SplitVertically<CR><C-w>l
```

**Comment by AlexPl292** [comment]
What about `nnoremap <leader>wS    <C-w>s<C-w>j`?

**Comment by marcoieni** [comment]
This is partially solved by #6 , the problem is that it does not work when you have more than two windows.

## Example

![2019-12-23_1657x263](https://user-images.githubusercontent.com/11428655/71352362-69d29b00-2576-11ea-97df-ba1e26f071d0.png)

you press `SPC w V`

![2019-12-23_1658x270](https://user-images.githubusercontent.com/11428655/71352438-a0101a80-2576-11ea-8e2f-0a2207861ba8.png)

The expected behavior is that the focus goes to the window in the center.

This should be related to [this](https://youtrack.jetbrains.com/issue/VIM-1877) ideavim issue, so I close this issue, since I think we cannot solve it completely for now.

If someone has additional thoughts on this, please add a comment even if the issue is closed. Thanks.
